components/switch/stories/switch.test.js

```diff
@@ -22,9 +22,15 @@ export const SwitchGroup = Variants({
 			isChecked: true,
 		},
 		{
-			testHeading: "Checked and Disabled",
+			testHeading: "Checked and disabled",
 			isChecked: true,
 			isDisabled: true,
 		},
+		{
+			testHeading: "With longer label",
```

Comment:
Ok! I see that other components like Badge, Action Button, and Radio are doing this so I'll follow suit. I didn't name it "Truncation" though because it wraps--if I should for some reason, let me know!